verticapy/connection/external.py

```diff
@@ -34,10 +34,22 @@ def set_external_connection(cid: str, rowset: int = 500, symbol: str = "$") -> N
         A special character to identify the connection.
         One of the following:
         "$", "€", "£", "%", "@", "&", "§", "?", "!"
+
         For example, if the symbol is '$', you can call
         external tables with the input cid by writing
         $$$QUERY$$$, where QUERY represents a custom
         query.
+
+    Example
+    -------
+    .. code-block:: python
+
+        import verticapy as vp
+
+        # Set up a connection with a database with the alias "pgdb"
```

Comment:
PEP formatting. And write the comment outside of the function.

.. note:: When configuring an external connection, you'll need to assign a unique symbol to identify it. This symbol will subsequently allow you to extract data from the target database using the associated identifier.